docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -64,15 +64,46 @@ If you are reading this topic with the questions listed below, click them to jum
 
 4. Connect to Nebula Graph.
 
+    1. Start a Nebula Graph service in a Docker container
     ```bash
-    # Start a Nebula Graph service in a Docker container
     $ docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2-nightly
     ```
+    
+    > **Note**: Your local network  (nebula-docker-compose_nebula-net) may be different from the example above. Use the following command.
+    
+    ```bash
+    $ docker network  ls
+    NETWORK ID          NAME                             DRIVER              SCOPE
+    a74c312b1d16        bridge                           bridge              local
+    dbfa82505f0e        host                             host                local
+    ed55ccf356ae        nebuladockercompose_nebula-net   bridge              local
+    93ba48b4b288        none                             null                local
+
+    ```
 
+    2. Connect to Nebula Graph with Nebula Console
     ```bash
-    # Connect to Nebula Graph with Nebula Console
-    docker> nebula-console -u <user> -p <password> --address=graphd --port=9669
+    docker> nebula-console -u user -p password --address=graphd --port=9669
     ```
+    > **Note**: By default, the authentication is off. Check [TODO(doc)] to turn-on.
```

Comment:
> Check [TODO(doc)] to turn-on.

Check Enable authentication (TODO: doc) to turn it on.

Or: To turn it on, see Enable authentication (TODO: doc).